client/src/components/Tour/TourList.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <div class="ui-thumbnails">
+        <h2>Galaxy Tours</h2>
+        <p>
+            This page presents a list of interactive tours available on this Galaxy server. Select any tour to get
+            started (and remember, you can click 'End Tour' at any time).
+        </p>
+        <h4>Tours</h4>
+        <div v-if="error" class="alert alert-danger">{{ error }}</div>
+        <div v-else>
+            <DelayedInput class="mb-3" :query="search" :placeholder="searchTours" :delay="100" @change="onSearch" />
+            <div v-for="tour in tours" :key="tour.id">
+                <ul v-if="match(tour)" id="tourList" class="list-group">
+                    <li class="list-group-item">
+                        <a :href="`${root}tours/${tour.id}`"> {{ tour.name || tour.id }} </a>&nbsp;&minus;&nbsp;
+                        {{ tour.description }}
+                        <span v-for="(tag, index) in tour.tags" :key="index" class="badge badge-primary">
+                            {{ tag.charAt(0).toUpperCase() + tag.slice(1) }}
```

Comment:
Consider doing this with `text-transform` instead?

Reply:
Yes - this makes sense.